core/state/snapshot/disklayer.go

```diff
@@ -76,30 +76,28 @@ func (dl *diskLayer) Stale() bool {
 
 // Account directly retrieves the account associated with a particular hash in
 // the snapshot slim data format.
-func (dl *diskLayer) Account(hash common.Hash) (*types.SlimAccount, error) {
-	data, err := dl.AccountRLP(hash)
+func (dl *diskLayer) account(hash common.Hash, ignoreStale bool) (*types.SlimAccount, error) {
```

Comment:
thinking alternative names.. not sure they are better, but what do you think about  "evenIfStale" ?

Reply:
Sounds good to me, renamed